gallery_dl/extractor/kemono.py

```diff
@@ -6,27 +6,27 @@
 # it under the terms of the GNU General Public License version 2 as
 # published by the Free Software Foundation.
 
-"""Extractors for https://kemono.su/"""
+"""Extractors for https://kemono.cr/"""
 
 from .common import Extractor, Message
 from .. import text, util, exception
 from ..cache import cache, memcache
 import itertools
 import json
 
-BASE_PATTERN = r"(?:https?://)?(?:www\.|beta\.)?(kemono|coomer)\.(su|party)"
+BASE_PATTERN = r"(?:https?://)?(?:www\.|beta\.)?(kemono|coomer)\.(st|cr|su|party)"
```

Comment:
```suggestion
BASE_PATTERN = r"(?:https?://)?(?:www\.|beta\.)?(kemono|coomer)\.(cr|s[tu]|party)"
```

Matches `cr` first and "simplifies" the `s…` case